src/Containers/CircleTable.tsx

```diff
@@ -19,135 +31,241 @@ export interface ICircleTable {
      */
     occupancyStatus: occupancyStatusTypes;
     /**
-     * The seating/reservation time for the party at the table
+     * The size for the component relative to the parent
      */
-    reservationTime?: Date;
-}
-
-enum occupancyStatusTypes {
-    Vacant = 'Vacant',
-    Reserved = 'Reserved',
-    Occupied = 'Occupied',
+    relativeSize: number;
 }
 
 /**
  * Primary UI component for user interaction
  */
 export const CircleTable: React.FC<ICircleTable> = ({
     tableID = 'T1',
+    chairs = [],
     partyName = 'Null',
-    occupancyStatus = occupancyStatusTypes.Vacant,
+    occupancyStatus = 'Vacant',
+    relativeSize = 1.0,
     ...props
 }) => {
     const { colors } = useTheme();
 
     /**
-     * This function will determine what color should be the Status and ColorDiv
-     * and return hexadecimal color value
+     * Determines the correct color for Status and ColorDiv based on occupancyStatus
+     * and returns the hexadecimal color value as a string
      *
      * @return {string} - Hexadecimal color value
      */
-    function getOccupancyColor(): string {
+    const getOccupancyColor: getOccupancyColorType = () => {
         switch (occupancyStatus) {
-            case occupancyStatusTypes.Vacant:
+            case 'Vacant':
                 return colors.occupancyStatusColors.Vacant;
 
-            case occupancyStatusTypes.Reserved:
+            case 'Reserved':
                 return colors.occupancyStatusColors.Reserved;
 
-            case occupancyStatusTypes.Occupied:
+            case 'Occupied':
                 return colors.occupancyStatusColors.Occupied;
 
             default:
                 return '';
         }
```

Comment:
Should this live here or in the styled component? If this only affects styling it should be in styled components